core/src/com/unciv/logic/automation/NextTurnAutomation.kt

```diff
@@ -831,7 +830,7 @@ object NextTurnAutomation {
             when {
                 unit.baseUnit.isRanged() -> rangedUnits.add(unit)
                 unit.baseUnit.isMelee() -> meleeUnits.add(unit)
-                unit.hasUnique("Bonus for units in 2 tile radius 15%")
+                unit.isGreatPersonOfType("War")
```

Comment:
I'm not sure we need this distinction between generals she other civilian units at all